api-reference/docstubs/api/calendar-list-calendarview.md

```diff
@@ -0,0 +1,222 @@
+---
+title: "List calendarView"
+description: "Get the events from the calendarView navigation property."
+author: ""
+localization_priority: Normal
+ms.prod: ""
+doc_type: apiPageType
+---
+
+# List calendarView
+
+Namespace: microsoft.graph
+
+Get the events from the calendarView navigation property.
```

Comment:
Laura/Angela - would we call calenderView a navigation property or a relationship?
Should calenderView be in bold face?

Reply:
Yes, I don't think we should mention that navigation property part. And yes, technically the first mention would be a link to the resource topic, but if we can't do that we should apply bold to the resource names.

Reply:
@Lauragra So what should we put here?  Get the calendarView events?  calendarView is the name of the (navigation) property and it returns a collection of events.